src/DoctrineORMModule/Service/ConfigurationFactory.php

```diff
@@ -152,6 +152,10 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             $config->setSecondLevelCacheEnabled();
             $config->setSecondLevelCacheConfiguration($cacheConfiguration);
         }
+        
+        if($schemaFilter = $options->getSchemaFilter()) {
```

Comment:
I agree with you. I just thought it could be nice to adopt the same option name they've used in the symfony bundle :)